src/DotNet/Writer/MethodBodyWriterBase.cs

```diff
@@ -12,6 +12,8 @@ public abstract class MethodBodyWriterBase {
 		protected IList<Instruction> instructions;
 		/// <summary/>
 		protected IList<ExceptionHandler> exceptionHandlers;
+		/// <summary>The module context</summary>
+		protected ModuleContext context;
```

Comment:
Should be `readonly` and renamed to `contextOrNull`.

Is this field used anywhere? I couldn't find any uses.

Reply:
Looks like a leftover. Shouldn't be necessary.

Reply:
Seems like it's still left in the new code?